### PR TITLE
Fix job to update psalm baseline

### PR DESCRIPTION
**Pull request by kesselb**
The workflow to update the psalm baseline is broken for a while. I guess our token has no access rights to write to nextcloud-pr-bot anymore.

### CODE REVIEW

File: .github/workflows/update-psalm-baseline.yml [hunk 1]
```diff
@@ -25,18 +25,13 @@ jobs:
         run: composer i
       - name: Psalm
         run: composer run psalm -- --monochrome --no-progress --output-format=text --update-baseline
-      - name: Reset composer.json and composer.lock
```
Comment (kesselb) [inline .github/workflows/update-psalm-baseline.yml:28]: Christoph scoped the psalm dependency a while ago hence the composer reset is not required anymore.

**Comment by MichaIng** [comment]
Ah nice, I recognised this weekly job failing for a while but always forgot to open an issue about this 👍.

**Review by MichaIng — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by kesselb** [comment]
> does this need to be done on other branches to keep the baseline up to date there as well ?

ideally yes. right now we only run it for main.

**Comment by MichaIng** [comment]
I'm confused, the workflow still fails: https://github.com/nextcloud/server/actions/workflows/update-psalm-baseline.yml
Strange is that it still tries to push to a fork and has `push-to-fork: nextcloud-pr-bot/server` set. Viewing the file (from GitHub actions web UI) also shows the old version with this line present: https://github.com/nextcloud/server/actions/runs/1310389926/workflow
Hitting edit, then shows the correct version of master.

Is there something cached somewhere at the actions servers?

**Pull request merged**